core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -200,7 +200,7 @@ private[spark] class TaskSchedulerImpl(
         val execId = taskIdToExecutorId(tid)
         backend.killTask(tid, execId, interruptThread)
       }
-      tsm.abort("Stage %s cancelled".format(stageId))
+      tsm.abort("Stage %d was cancelled".format(stageId))
```

Comment:
Better still, change this to use string interpolation. But, printing an int as an int value, and print its toString as a string, gives the same result. This doesn't do anything so I don't think I'd bother with this.